Networking/NetworkingTests/Mapper/ProductMapperTests.swift

```diff
@@ -253,7 +271,7 @@ private extension ProductMapperTests {
     /// Returns the ProductMapper output upon receiving `product`
     ///
     func mapLoadProductResponse() -> Product? {
-        return mapProduct(from: "product")
+        return mapProduct(from: "product-downloadable")
```

Comment:
Yes, you are right. Initially, I tried with changing the existing `product .json` file but it broke a few existing test cases, so I decided not to touch the existing ones and add a new file. But since you questioned this choice, now I think it would be better to do what you suggested and fix the failing test cases.

Reply:
It's totally fine to update the tests cases 👍

Reply:
Already updated.  🙂